src/new-components/base-button/base-button.module.css

```diff
@@ -119,6 +119,10 @@
     --reactist-spinner-fill: var(--reactist-btn-idle-fill);
 }
 
+.baseButton.shape-rounded {
+    border-radius: 1000px;
```

Comment:
it doesn't change anything, just curious why not to use `border-radius: 100%`?

Reply:
I've never understood why, but that has an unintended effect:

<img width="376" alt="CleanShot 2022-07-04 at 20 05 28@2x" src="https://user-images.githubusercontent.com/15199/177227046-f99b0d3b-cadf-4dc7-9e04-b98ac8c935ca.png">

I've always had to use a very large number when wanting a full round border radius.

Reply:
We could reserve this one for `shape="fish"`?